src/styles/blocks/calendar.scss

```
@@ -0,0 +1,47 @@
+@import "../utils/_variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $block-size + $gap * 6;
+  margin: 0 auto;
+  padding-top: 40px;
+  gap: $gap;
+
+  @each $day, $number in $days-of-week {
+    &--start-day-#{$day} :first-child {
+      margin-left: $number * $block-size + ($gap * $number);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    position: relative;
+    width: $block-size;
+    height: $block-size;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: transform linear 0.5s;
```

create a variable for this & replace in all cases
```suggestion
    transition: transform linear $transitionDuration;
```